src/main/java/net/dv8tion/jda/api/utils/messages/MessageCreateBuilder.java

```diff
@@ -232,6 +242,11 @@ public MessageCreateData build()
         List<LayoutComponent> components = new ArrayList<>(this.components);
         AllowedMentionsData mentions = this.mentions.copy();
 
+        if(silent)
+            messageFlags |= Message.MessageFlag.NOTIFICATIONS_SUPPRESSED.getValue();
+        else
+            messageFlags &= ~Message.MessageFlag.NOTIFICATIONS_SUPPRESSED.getValue();
```

Comment:
This should probably be done in the setter instead, no need to keep track of the same information in 2 fields.

Reply:
Done.